fuel/transformers/__init__.py

```diff
@@ -118,8 +136,15 @@ def get_data(self, request=None):
 
 
 @add_metaclass(ABCMeta)
-class SingleMapping(Transformer):
-    """Applies a single mapping to multiple sources.
+class SourcewiseTransformer(Transformer):
```

Comment:
I would say that `transform_any_source` kinda breaks symmetry with `transform_source_batch` and `transform_source_example`.